README.md

```diff
@@ -123,18 +123,14 @@ Ginkgo adds the following additional switches to control what is being built:
 
     *   `Auto`
     *   `Kepler`, `Maxwell`, `Pascal`, `Volta`
-    *   `COMPUTE`, `COMPUTE(CODE)`, `(CODE)`, `MaxPTX`
-    *   `Off`
-
-    `Auto` will automatically detect the present CUDA-enabled GPU 
-    architectures in the system.
-    `Kepler`, `Maxwell`, `Pascal` and `Volta` will add flags for all
-    architectures of that particular NVIDIA GPU generation. `COMPUTE` and `CODE`
-    are placeholders that should be replaced with compute and code numbers (e.g.
-    for `compute_70` and `code_70` `COMPUTE` and `CODE` should be replaced with
-    `70`. `MaxPTX` will select the latest architecture supported by the
-    compiler. `Off` will not select any architectures and compile with NVCC's
-    default settings. Default is `Auto`.
+    *   `CODE`, `CODE(COMPUTE)`, `(COMPUTE)`
+
+    `Auto` will automatically detect the present CUDA-enabled GPU architectures
+    in the system. `Kepler`, `Maxwell`, `Pascal` and `Volta` will add flags for
+    all architectures of that particular NVIDIA GPU generation. `COMPUTE` and
+    `CODE` are placeholders that should be replaced with compute and code
+    numbers (e.g.  for `compute_70` and `sm_70` `COMPUTE` and `CODE` should be
+    replaced with `70`. Default is `Auto`.
```

Comment:
Maybe you could somewhere add a link to the CAS repository, also indicating the users if they want more details they can find it there.